server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -98,6 +98,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion INITIAL_ELASTICSEARCH_8_17_4 = def(8_797_0_04);
     public static final TransportVersion INITIAL_ELASTICSEARCH_8_17_5 = def(8_797_0_05);
     public static final TransportVersion V_8_18_0 = def(8_840_0_00);
+    public static final TransportVersion V_8_19_0 = def(8_900_0_00);
```

Comment:
Adding a new transport version is not necessary for this PR, but for future notice this would not have been the right thing to do. First we're adding a new version for 8.19 while developing for 9.3+. Going forward please reference the updated guidance for versioning here: https://github.com/elastic/elasticsearch/blob/main/docs/internal/Versioning.md

For this PR however, please remove.